src/vs/workbench/contrib/output/browser/outputServices.ts

```diff
@@ -145,12 +150,21 @@ export class OutputService extends Disposable implements IOutputService, ITextMo
 		return null;
 	}
 
-	async showChannel(id: string, preserveFocus?: boolean): Promise<void> {
+	async showChannel(id: string, preserveFocus?: boolean, force?: boolean): Promise<void> {
 		const channel = this.getChannel(id);
 		if (this.activeChannel?.id !== channel?.id) {
 			this.setActiveChannel(channel);
 			this._onActiveOutputChannel.fire(id);
 		}
+		if (channel && !force && !this.viewsService.isViewVisible(OUTPUT_VIEW_ID) && this.configurationService.getValue('output.showQuietly')) {
+			this.notificationService.prompt(
```

Comment:
Not to convinced about the silent notifications approach. What about having a label in the status bar mentioning that a channel is trying to be revealed and when licking on it would reveal it. In that case there would at most be one indication ever showing.

Reply:
Done